pkg/theia/commands/anomaly_detection_run_test.go

```diff
@@ -89,15 +127,18 @@ func TestAnomalyDetectionRun(t *testing.T) {
 			cmd.Flags().Bool("use-cluster-ip", true, "")
 			cmd.Flags().String("algo", "ARIMA", "")
 			cmd.Flags().String("start-time", "2006-01-02 15:04:05", "")
-			cmd.Flags().String("end-time", "2006-01-03 15:04:05", "")
+			cmd.Flags().String("end-time", "2006-01-03 16:04:05", "")
```

Comment:
whats the reason for extending by 1 hour?

Reply:
because the end time should be after the start-time,

Reply:
If you know the execution time for all the times, lets set that value instead of waiting for such a long time

Reply:
This is a table entry, it has nothing to do with how long the job will run for